compiler/rustc_codegen_ssa/src/back/metadata.rs

```diff
@@ -379,6 +379,30 @@ pub(super) fn elf_e_flags(architecture: Architecture, sess: &Session) -> u32 {
             };
             e_flags
         }
+        Architecture::PowerPc64 => {
+            const EF_PPC64_ABI_UNKNOWN: u32 = 0;
+            const EF_PPC64_ABI_ELF_V1: u32 = 1;
+            const EF_PPC64_ABI_ELF_V2: u32 = 2;
+            
+            match sess.target.options.abi.as_ref() {
```

Comment:
None of the PowerPC targets indicate their ABI version that I'm aware of.

Reply:
That can be changed, there's only a handful of them.